packages/interactive_media_ads/lib/src/android/android_ads_loader.dart

```diff
@@ -79,13 +80,18 @@ base class AndroidAdsLoader extends PlatformAdsLoader {
   }
 
   @override
-  Future<void> requestAds(AdsRequest request) async {
+  Future<void> requestAds(PlatformAdsRequest request) async {
```

Comment:
The platform implementations haven't been exposed yet, so this shouldn't be a breaking change.